client/blocks/product-purchase-features-list/advertising-removed.jsx

```diff
@@ -22,10 +22,12 @@ export default localize( ( { isBusinessPlan, translate } ) => {
 					isBusinessPlan
 						? translate( 'All WordPress.com advertising has been removed from your site.' )
```

Comment:
Hi! I've found a possible matching string that has already been translated **18** times:
`translate( 'All WordPress.com advertising has been removed from your site. Upgrade to remove the WordPress.com footer credit.' )` *ES Score: 15*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).